doc/classification.xml

```diff
@@ -0,0 +1,540 @@
+<?xml version="1.0"?>
+<?mso-application progid="Excel.Sheet"?>
+<Workbook xmlns="urn:schemas-microsoft-com:office:spreadsheet"
+ xmlns:o="urn:schemas-microsoft-com:office:office"
+ xmlns:x="urn:schemas-microsoft-com:office:excel"
+ xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet"
+ xmlns:html="http://www.w3.org/TR/REC-html40">
+ <DocumentProperties xmlns="urn:schemas-microsoft-com:office:office">
+  <Author>Reuven Weiser</Author>
+  <LastAuthor>Microsoft Office User</LastAuthor>
+  <Created>2014-01-22T11:53:35Z</Created>
+  <LastSaved>2023-07-07T08:10:19Z</LastSaved>
+  <Version>16.00</Version>
```

Comment:
can we have a _pure_ XML file? Seeing Excel related stuff in a github repo is a bit disturbing

Reply:
Checking on that w/ Zoomin.